src/domain/event/EventUtils.ts

```diff
@@ -403,3 +409,10 @@ export const getAudienceAgeText = (
   }`;
   return `${ageLimit} -${t('event.info.age')}`;
 };
+
+export const getEventTypeByEventTypeId = (
+  eventTypeId: EventTypeId
+): EventType =>
+  Object.keys(EVENT_TYPE_TO_ID).find(
+    (eventType) => EVENT_TYPE_TO_ID[eventType as EventType] === eventTypeId
+  ) as EventType;
```

Comment:
Hmm I don't understand why this is needed. Is it to exclue 'All' type or something?

Reply:
Event object that is typed with EventFields now has a field called typeId, which is 'General' or 'Course'. In some cases, e.g when I'm fetching a list of other event times, some of the events might have a different type than the related event. Instead of hard coding and typing the EventType for the event, I would like to use the one stored in database and there I need that `getEventTypeByEventTypeId` -function, which converts 'General' to 'event' and ''Course' to a 'course'. Like that you can have a general event that has some related courses and when the course is clicked, user will be taken to a course section instead of event section - it will change the url to the right one!

Reply:
Oh yeah I was little bit blind here, didn't notice that this returns different type than takes as a parameter